6.X/6.14.0/README.md

```diff
@@ -58,7 +58,7 @@ This install assumes the Kubernetes cluster has network access to pull images fr
     provided with your Sysdig purchase confirmation mail
   - [`sysdig.platformAuditTrail.enabled`](docs/02-configuration_parameters.md#sysdigplatformAuditTrailenabled):
     Set this parameter to `true` if you would like to use Sysdig Platform Audit.
-  - [`sysdig.secure.events.audit.config.store.ip.enabled`](docs/02-configuration_parameters.md#sysdigsecureeventsauditconfigstoreipenabled):
+  - [`sysdig.secure.events.audit.config.store.ip`](docs/02-configuration_parameters.md#sysdigsecureeventsauditconfigstoreipenabled):
```

Comment:
@benpantrey 
I think you need to fix also the anchor:
`sysdigsecureeventsauditconfigstoreipenabled` is now https://github.com/draios/onprem-install-docs/pull/133/files#diff-e97ca1c974616cf32c896d244095f4d1474c884a13dbda9b75db9eae39b2ef8aR1336 and hence I think it's:
`sysdigsecureeventsauditconfigstoreip`